cli/cmd/org/members.go

```diff
@@ -1,20 +1,218 @@
 package org
 
 import (
-	"fmt"
-
+	"errors"
+	"github.com/rilldata/rill/admin/client"
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
 	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
 	"github.com/spf13/cobra"
 )
 
 func MembersCmd(cfg *config.Config) *cobra.Command {
 	membersCmd := &cobra.Command{
 		Use:   "members",
 		Short: "Members",
-		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("not implemented")
+	}
+	membersCmd.AddCommand(ListMembersCmd(cfg))
+	membersCmd.AddCommand(AddCmd(cfg))
+	membersCmd.AddCommand(RemoveCmd(cfg))
+	membersCmd.AddCommand(SetRoleCmd(cfg))
+	membersCmd.AddCommand(LeaveOrgCmd(cfg))
+
+	membersCmd.PersistentFlags().StringVar(&cfg.OrgOverride, "org", "", "Organization Name")
```

Comment:
This will clash with another PR, probably safest to remove from this PR and rely on default